test/extensions/retry/host/previous_hosts/integration_test.cc

```diff
@@ -0,0 +1,107 @@
+#include "common/protobuf/message_validator_impl.h"
+#include "common/protobuf/utility.h"
+
+#include "test/integration/http_integration.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Retry {
+namespace Host {
+namespace {
+
+class PrevioustHostsIntegrationTest : public testing::Test, public HttpIntegrationTest {
+public:
+  PrevioustHostsIntegrationTest()
+      : HttpIntegrationTest(Http::CodecClient::Type::HTTP2, Network::Address::IpVersion::v4) {}
+
+  void initialize() override {
+    // Add the retry configuration to a new virtual host.
+
```

Comment:
remove extra whitespace, and maybe move this comment down to the new vhost config?

Reply:
mind removing the newline before setDeterministic too?